pom.xml

```diff
@@ -6095,12 +6083,6 @@
           <artifactId>exec-maven-plugin</artifactId>
           <version>1.5.0</version>
         </plugin>
-        <plugin>
-          <groupId>org.sonatype.plugins</groupId>
-          <artifactId>nxrm3-maven-plugin</artifactId>
-          <version>1.0.2</version>
-          <extensions>true</extensions>
-        </plugin>
```

Comment:
The global plugin declaration was fine here.

Reply:
everything is no in the profile 👍